app/src/main/java/com/example/androidacademyhw/data/JsonLoad.kt

```diff
@@ -0,0 +1,110 @@
+package com.example.androidacademyhw.data
+
+import android.content.Context
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+
+
+private val jsonFormat = Json { ignoreUnknownKeys = true }
+
+@Serializable
+private class JsonGenre(val id: Int, val name: String)
+
+@Serializable
+private class JsonActor(
+    val id: Int,
+    val name: String,
+    @SerialName("profile_path")
+    val profilePicture: String
```

Comment:
IMHO it would be great and safe to annotate all the variables. 
Without it, there is a risk that someone will rename the variable and forgot to annotate it.

Reply:
Agree. The task was to copy this file from android academy repository. 
Can we leave it like that, since it will no longer be actual when we implement network requests in hw 8?

Reply:
Yeap